app/src/main/java/com/example/dogbook/MainActivity/ViewHolders/PostViewHolder.java

```diff
@@ -0,0 +1,59 @@
+package com.example.dogbook.MainActivity.ViewHolders;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.example.dogbook.MainActivity.Models.Post;
+import com.example.dogbook.R;
+import com.parse.ParseFile;
+
+public class PostViewHolder extends RecyclerView.ViewHolder {
+
+
+    ImageView ivProfilePicture;
+    ImageView ivPostPicture;
+    TextView tvUsername;
+    TextView tvOwner;
+    TextView tvCaption;
+    Context context;
+
+    public PostViewHolder(@NonNull View itemView, Context context) {
+        super(itemView);
+        ivProfilePicture = itemView.findViewById(R.id.ivProfilePicture);
+        ivPostPicture = itemView.findViewById(R.id.ivPostPicture);
+        tvUsername = itemView.findViewById(R.id.tvUsername);
+        tvOwner = itemView.findViewById(R.id.tvOwner);
+        tvCaption = itemView.findViewById(R.id.tvCaption);
+        this.context = context;
+    }
+
+    public void bind(Post post) {
+        ParseFile profilePicture = post.getAuthor().getParseFile("profilePicture");
```

Comment:
Can save this into a static constant -- likely makes sense to live in the Post file